app/javascript/src/Policies/components/CustomPolicy.jsx

```diff
@@ -57,23 +57,38 @@ function CustomPolicyForm ({policy, onChange, win = window}: {policy: Policy, on
   }
 
   return (
-    <form action={action} method="post">
-      <label>
-        Directory:
-        <input type="text" name="directory" value={policy.directory} onChange={onChange} disabled={!isNewPolicy} />
-      </label>
+    <form action={action} method="post" className="formtastic">
+      <br/>
+      <fieldset className="inputs">
+        <legend>Place of the Custom Policy Code</legend>
+        <ol>
+          <li>
+            <label>Path to where the Policy is located in your APIcast installation</label>
+            <input placeholder="policy-name/1.0.0/" type="text" name="directory" value={policy.directory} onChange={onChange} />
+            <p className="inline-hints">The path to your custom policy relative to <code>APICAST_DIR/policies/</code></p>
```

Comment:
On the rest of this section, it's referred as `Custom Policy` starting with capital letters . Not here tho. I'm not sure how it's supposed to be, but would be good at least to keep it consistent.

Reply:
yep, thanks!